EventsExpress/ClientApp/src/containers/event-item-view.js

```diff
@@ -33,28 +35,31 @@ class EventItemViewWrapper extends Component{
 
     render(){   
         const { isPending } = this.props.event;
-  
         return isPending
             ? <Spinner />
-            : <EventItemView 
+            : <EventItemView
+                inventories={this.props.inventories} 
                 event={this.props.event}
                 match={this.props.match} 
                 onLeave={this.onLeave} 
                 onJoin={this.onJoin}
                 onCancel={this.onCancel}
                 onApprove={this.onApprove}
-                current_user={this.props.current_user} 
+                current_user={this.props.current_user}
+                get_inventories_by_event_id={this.props.get_inventories_by_event_id} 
             />
     }
 }
 
 const mapStateToProps = (state) => ({
     event: state.event, 
-    current_user: state.user
+    current_user: state.user,
+    inventories: state.inventories
```

Comment:
Please consider not passing inventories from here, but connecting it directly in the component.